modules/packages/Buffers.chpl

```diff
@@ -117,7 +117,7 @@ module Buffers {
 
    */
   pragma "ignore noinit"
-  record bytes {
+  record byteBuffer {
```

Comment:
Quick check: Was @mppf good with this name change?  (as the original author of this file)

Reply:
Yes. On an offline discussion before I opened this PR, I brought this issue up and this was the best option we could come up with. I will check with him once again before this is merged.

Reply:
Yes it is fine for the moment.